tests/impl.cpp

```diff
@@ -3429,7 +3429,15 @@ result_t test_mm_srli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int64_t *_a = (const int64_t *) impl.mTestIntPointer1;
+    const int count = 3;
```

Comment:
The last parameter of `_mm_slli_epi64` is `imm8`, and you took one numeric constant `3`.  You can add comments such as `const int count = 3; /* imm8 <= 63 */`.
Even better, you can do another test for `imm8 > 63`.